Neos.Kickstarter/Classes/Service/GeneratorService.php

```diff
@@ -553,13 +582,8 @@ protected function generateFile($targetPathAndFilename, $fileContent, $force = f
             \Neos\Utility\Files::createDirectoryRecursively(dirname($targetPathAndFilename));
         }
 
-        if (substr($targetPathAndFilename, 0, 11) === 'resource://') {
-            list($packageKey, $resourcePath) = explode('/', substr($targetPathAndFilename, 11), 2);
-            $relativeTargetPathAndFilename = $packageKey . '/Resources/' . $resourcePath;
-        } elseif (strpos($targetPathAndFilename, 'Tests') !== false) {
-            $relativeTargetPathAndFilename = substr($targetPathAndFilename, strrpos(substr($targetPathAndFilename, 0, strpos($targetPathAndFilename, 'Tests/') - 1), '/') + 1);
-        } else {
-            $relativeTargetPathAndFilename = substr($targetPathAndFilename, strrpos(substr($targetPathAndFilename, 0, strpos($targetPathAndFilename, 'Classes/') - 1), '/') + 1);
+        if (strpos($targetPathAndFilename, FLOW_PATH_PACKAGES) === 0) {
```

Comment:
A little note about this change.

I had a discussion with @kitsunet about a problem that i had, while working on my `./flow kickstart:nodemigration` feature. The problem was that the output in my `generateNodeMigration()` method looked like this:

```
Created .../Test20220906183945.yaml
```

The full path to the file was missing. This was because the `generateFile()` method could only handle Resources, Tests and Classes. We refactored the `generateFile()` method so that it works the same as before. But now it also can handle the full path to a migration created with the `./flow kickstart:nodemigration` command:

```
Created .../Sites/Neos.Demo/Migrations/ContentRepository/Test20220907193517.yaml
Your node migration has been created successfully.
```